### ISSUE TITLE
ISO/IEC 27001:2013 IT Policies & Procedures

### ISSUE THREAD

**Issue by nelsonic** [OxfordAbstracts/information-technology-security-policies#1]
The [**ISO 27001 _Standard_**](http://www.iso.org/iso/iso27001) defines requirements for the **confidentiality**, **integrity** and **availability** of information systems. 

+ What is ISO 27001?  (_video intro_): https://youtu.be/AzSJyfjIFMw
+ ISO 27001 Certification: https://youtu.be/mMmpAwmXRNU
+ Implementation Guide: http://www.bsigroup.com/Documents/iso-27001/resources/iso-iec-27001-implementation-guide-SG-web.pdf

To ensure that both _we_ (_the OA team_) and our _stakeholders_ (_people who use the app for their business-critical information_) have the _confidence_ in our IT systems we need answer all **133 questions** in the risk assessment.

_Annoyingly_ the **Standard** is not _freely_ available, instead we have to _pay_ to download it!

**Comment by nelsonic** [comment]
A bit of googling and found: https://trofisecurity.com/assets/img/iso27001-2013.pdf

**Comment by Boatlaunch101** [comment]
Very handy and I see where the Elsevier questions have come from now, i.e. ISO27001.
By filling out the Elsevier SIG sheet we will have answered most of the ISO27001 questions.

**Comment by nelsonic** [comment]
@Boatlaunch101 indeed. that's the plan. by answering the questions from the perspective of **ISO/IEC 27001:2013** we can (_optionally_) apply for _certification_ in 2017 which will _further improve_ "_credibility_" of the product. (_though it does take time/money to get certified..._)
